lib/common/models/collection.rb

```diff
@@ -83,7 +84,13 @@ def finder(object, search)
         predicates.all? do |operator, expected_value|
           op = OPERATIONS_MAP.fetch(operator)
           mock_comparator_object.send("#{attribute}=", expected_value)
-          actual_value.send(op, mock_comparator_object.send(attribute))
+
+          if op == :match
+            rexp = Regexp.escape(Regexp.escape(mock_comparator_object.send(attribute)))
```

Comment:
Is it doing this regex at the DB level or after we get a set of results back and then just pruning it? If it's at the DB level, I think it would be potentially too expensive. If it's after we have results, can't we just do the regex on the client side?